lib/api/widget/functions.php

```diff
@@ -25,21 +25,22 @@
  *
  * @since 1.0.0
  *
- * @param array $args {
- *     Optional. Arguments used by the widget area.
+ * @param array $args                       {
```

Comment:
It's a formatting issue with PhpStorm that I'm working on to give us a standard one-click approach to reformat and comply.

Reviewing the [WPCS document for array documentation](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-1-parameters-that-are-arrays), they show this:

```
/**
 * Summary.
 *
 * Description.
 *
 * @since x.x.x 
 *
 * @param type  $var Description.
 * @param array $args {
 *     Optional. An array of arguments.
 *
 *     @type type $key Description. Default 'value'. Accepts 'value', 'value'.
 *                     (aligned with Description, if wraps to a new line)
 *     @type type $key Description.
 * }
 * @param type  $var Description.
 * @return type Description.
 */
```